### ISSUE TITLE
failed when `pc run`

### ISSUE THREAD

**Issue by LayGPark** [pynecone-io/pynecone#897]
**Describe the bug**
I just only setup ubuntu docker.
and install python (`apt install -y python3 python3-pip`)
and install nodejs(`wget https://nodejs.org/dist/v18.16.0/node-v18.16.0-linux-x64.tar.xz`)
and setup virtualenv (
  ```
pip install virtualenv
virtualenv .
source ./bin/activate
pip install pynecone
  ```
).

I think I've done my job.


<img width="593" alt="image" src="https://user-images.githubusercontent.com/13059827/235031239-ae586a5e-44d7-4c5d-b9af-cc23406fce86.png">

but, result are always as below.

**Screenshots**
```
[01:07:09] Initializing the web directory.                                                                                                                                                                                                                                                                                              console.py:30
           Initializing the app directory.                                                                                                                                                                                                                                                                                              console.py:30
[01:07:10] Finished Initializing: pyneplay                                                                                                                                                                                                                                                                                              console.py:30
(pyneplay) ➜  pyneplay pc run
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── Starting Pynecone App ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── Installing frontend packages ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
bun install v0.5.9 (f3ab445c)
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── App Running ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
$ next dev
/usr/bin/bash: line 1: next: command not found
error: script "dev" exited with code 127
```

** Specifics (please complete the following information):**
 - Python Version: 3.10.6
 - Pynecone Version: 0.1.27
 - OS: ubuntu
 - Browser (Optional):
 
**Additional context**


I need help, please.

**Comment by Alek99** [comment]
That's odd I'm not getting this error, anyone else getting this?

Ill try on a docker

**Comment by Hspix** [comment]
new version 0.1.27 maybe some problems ?  
0.1.24 is ok

**Comment by Alek99** [comment]
Ok thanks for taking a look we will try and release a fix tomorrow.

**Comment by Alek99** [comment]
> **Describe the bug** I just only setup ubuntu docker. and install python (`apt install -y python3 python3-pip`) and install nodejs(`wget https://nodejs.org/dist/v18.16.0/node-v18.16.0-linux-x64.tar.xz`) and setup virtualenv (
> 
> ```
> pip install virtualenv
> virtualenv .
> source ./bin/activate
> pip install pynecone
> ```
> 
> ).
> 
> I think I've done my job.
> 
> <img alt="image" width="593" src="https://user-images.githubusercontent.com/13059827/235031239-ae586a5e-44d7-4c5d-b9af-cc23406fce86.png">
> 
> but, result are always as below.
> 
> **Screenshots** If applicable, add screenshots to help explain your problem.
> 
> ```
> [01:07:09] Initializing the web directory.                                                                                                                                                                                                                                                                                              console.py:30
>            Initializing the app directory.                                                                                                                                                                                                                                                                                              console.py:30
> [01:07:10] Finished Initializing: pyneplay                                                                                                                                                                                                                                                                                              console.py:30
> (pyneplay) ➜  pyneplay pc run
> ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── Starting Pynecone App ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
> ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── Installing frontend packages ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
> bun install v0.5.9 (f3ab445c)
> ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── App Running ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
> $ next dev
> /usr/bin/bash: line 1: next: command not found
> error: script "dev" exited with code 127
> ```
> 
> ** Specifics (please complete the following information):**
> 
> * Python Version: 3.10.6
> * Pynecone Version: 0.1.27
> * OS: ubuntu
> * Browser (Optional):
> 
> **Additional context** Add any other context about the problem here.
> 
> I need help, please.

What's your base docker we can try and recreate the error and see what's causing it

**Comment by picklelo** [comment]
The following Dockerfile works for me:

```dockerfile
# A very simple example of a Dockerfile.
FROM ubuntu

# Install dependencies.
RUN apt-get update && apt-get install -y \
    python3-dev \
    python3-pip \
    nodejs \
    curl \
    unzip

# Install pynecone
RUN pip install pynecone

# Run the example app.
RUN mkdir /test; cd /test; pc init;

WORKDIR /test
```

Then build as

```bash
docker build . -t pctest
```

And run as
```bash
docker run -p 3000:3000 -p 8000:8000 pctest pc run
```

Let me know if that works, may have to update existing Dockerfiles.

**Comment by LayGPark** [comment]
To. @Alek99 
ubuntu. same version as @picklelo 
because, i started trying this week.

**Comment by picklelo** [comment]
@LayGPark Can you see if running the above commands works for you? I just tried on the `ubuntu` docker image. Or do you still see the same error

**Comment by LayGPark** [comment]
@picklelo 
Thank you for reply!
I need some times ;) (docker volume issue...)
anyway, result is changed.
```
 $ docker run -p 3049:3000 -p 8049:8000 pctest pc run
──────────────────────────── Starting Pynecone App ─────────────────────────────
───────────────────────── Installing frontend packages ─────────────────────────
bun install v0.5.9 (f3ab445c)
───────────────────────────────── App Running ──────────────────────────────────
$ next dev
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
event - compiled client and server successfully in 4.4s (760 modules)
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry
```


<img width="739" alt="image" src="https://user-images.githubusercontent.com/13059827/235080309-c3945405-2470-404b-95a6-2cf250c1c719.png">


add 2 questions

1) why success this docker? -_-? same ubuntu... virtualenv issue...?

2) where use 8000?

**Comment by milochen0418** [comment]
> That's odd I'm not getting this error, anyone else getting this?
> 
> Ill try on a docker


I find the out the same issue. 
https://github.com/pynecone-io/pynecone-examples/issues/56
I will make a new nodejs  environment to test more and to see where the bug it is.

**Comment by r0b2g1t** [comment]
The main problem is that Pynecone run for environment prod doesn't download the bun dependencies. That was fixed in PR #887 . The Dockerfile will run Pynecone in prod environment and will download everything with bun with the new version.

**Comment by r0b2g1t** [comment]
I have fixed the Dockerfile by adding a manual run install after the ```pc init``` which installs the bun packages and provides a clean image. PR #911

**Comment by Alek99** [comment]
Thanks @r0b2g1t will review this pr tomorrow!

**Comment by picklelo** [comment]
This should be fixed by #938
